update german lang-file

PhoenixSk4 opened the pull request:
This is a complete german translation from the (latest) en_en.lang file. I made some comments while translating, search for "#-" to find all comments.

## What
<!--This section describes what this PR is about. It should be a clear and concise description concerning what this PR is for, why this PR is needed, and why it should be accepted.-->
Full German translation
<!--Linking an issue can be used alternatively to writing a description.-->

## Implementation Details
<!--Any implementations in this PR that should be carefully looked over, or that could/should have alternate solutions proposed.-->
Some comments, search for "#-" to find all my comments

## Outcome
<!--A short description of what this PR added/fixed/changed/removed.-->
<!--For correct linking of issues please use any of the Closes/Fixes/Resolves keywords. Example: When a PR is fixing a bug use "Fixes: #number-of-bug"-->
Adds german translation

## Additional Information
<!--This section is for screenshots to demonstrate any GUI or rendering changes, or any other additional information that reviewers should be aware of.-->
Sometimes the translation is not completly clear but understandable. (Notable ones have been commented)

## Potential Compatibility Issues
<!--This section is for defining possible compatibility issues. It must be used when there item/block/material/machine changes, or recipe changes.-->
German symbols like äöüß? 

<!--**Please fill in as much useful information as possible. Also, please remove all unused sections, including this and the other explanations.**-->

PhoenixSk4 committed:
update gernab lang-file

This is a complete german translation from the (latest) en_en.lang file. I made some comments while translating, search for "#-" to find all comments.

trainvoi commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2760 @@
+susy.material.stellite_j=Stellite J
+susy.material.stellite_6=Stellite 6
+susy.material.hsla_980_x=HSLA 980X
+#- Wait, you can eat stainless steel? -PhoenixSky
```
it's steel that doesn't leach toxins to food i guess

PhoenixSk4 replied:
Yes, I know, made some yanky comments that I actually wanted to remove, but forgot. Sorry about that, these can be removed and were just there to keep me a little entertained in between. 😭

trainvoi commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2760 @@
+recipemap.cooling_tower.name=Kühlturm
+recipemap.gas_turbine_new.name=Gasturbine
+#- You know the drill by now, please help and make suggestions - PhoenixSky
+recipemap.quencher.name=Abschrecker
```
this isn't a good translation of quencher....

PhoenixSk4 replied:
Yep, I was unsure about a 100% translation. After reviewing again, Abschreckanlage would fit better.

I guess from a little head review, quenching would be translated as Abschrecken. Abschreckbecken would be the correct technical term. Sorry 👍

trainvoi commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2760 @@
+metaitem.crucible.quartz.name=Quarz-Tiegel
+metaitem.crucible.graphite.name=Graphit-Tiegel
+
+#- Again I am unsure of the real translation of Sponge
```
have you try googling titanium sponge in german

PhoenixSk4 replied:
Yes, it is the correct translation, but sounds a little off as a native speaker, was referring to a more "fluent" translation.

PhoenixSk4 committed:
Update de_de.lang

PhoenixSk4 commented:
Removed unnecessary comments that I forgot to remove, reviewed that Schwamm is the correct translation. Abschrecker is not the correct technical translation, changed to "Abschreckbecken".

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+susy.material.potassium_ethyl_xanthate=Kaliumethylxanthogenat
+susy.material.potassium_ethyl_xanthate_solution=Kaliumethylxanthogenatlösung
+susy.material.silica_gel=Silicagel
+susy.material.wet_silica_gel=Feuchtes Silicagel
```
Ich würde es "nass" statt feucht nennen

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+susy.material.quenched_atmospheric_oil_residue=Abgeschreckter atmosphärischer Ölrückstand
+susy.material.quenched_vacuum_oil_residue=Abgeschreckter Vakuum-Ölrückstand
+susy.material.light_gas_oil=Leichtgasöl
+susy.material.slurry_oil=Slurryöl
```
"Slurry" ist kein deutsches word, oder?
Wie wäre Schlammöl ?

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+# GregTech RecipeMaps
+
+recipemap.fluid_decompressor.name=Fluid (De-)Kompression
+recipemap.fluid_compressor.name=Fluidkompression
```
Kompressor oder Gaskompressor passt besser

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+
+# GregTech RecipeMaps
+
+recipemap.fluid_decompressor.name=Fluid (De-)Kompression
```
Dekomprimierer

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+recipemap.quencher.name=Abschreckbecken
+recipemap.steam_turbine_new.name=Dampfturbine
+recipemap.railroad_engineering_station.name=Eisenbahn-Ingenieurstation
+recipemap.dryer.name=Trocknung
```
Trockner

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+recipemap.steam_turbine_new.name=Dampfturbine
+recipemap.railroad_engineering_station.name=Eisenbahn-Ingenieurstation
+recipemap.dryer.name=Trocknung
+recipemap.sintering_oven.name=Sintern
```
Sinterofen oder Sinteröfen

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+recipemap.coking_tower.name=Verkokungsturm
+recipemap.ion_exchange_column.name=Ionenaustausch
+recipemap.smoke_stack.name=Schornstein
+recipemap.gravity_separator.name=Schwerkrafttrennung
```
vielleicht passt Schwerkraftabscheider besser

Rick8484 commented on resources/langfiles/lang/de_de.lang:
```diff
@@ -1,9 +1,2737 @@
+recipemap.catalytic_reformer_recipes.name=Katalytisches Reforming
+recipemap.coking_tower.name=Verkokungsturm
+recipemap.ion_exchange_column.name=Ionenaustausch
+recipemap.smoke_stack.name=Schornstein
```
Liebe "Schlot" nennen

PhoenixSk4 committed:
Update de_de.lang

Removed duplicate line of laser engraver, updated translation as suggested by rick. Fixed two translation errors from chemistry (depleted uranium dioxide, carbon tetrafluride). Fixed minor formatting (mainly two much empty lines.

PhoenixSk4 committed:
Update de_de.lang 

Removal of uneccesarry comments

Rick8484 commented:
Seems to be merge able.

Rick8484 reviewed (approved):
Seems to be merge able but more review is always better.

bruberu reviewed (approved):
By this point, I think it will be ready to be merged by the time of the next update.

loxoDev committed:
Update de_de.lang

- properly use hyphen
- translate xyz leach solution with XYZ-Laugungs-Lösung (its

loxoDev commented:
Please wait until I finish reviewing.
